2048Solution/main/java/models/GameBoard.java

```diff
@@ -0,0 +1,70 @@
+package main.java.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameBoard {
+
+
+    private int size;
+    private int[][] board;
+    private int rowToBeFilled;
+    private int columnToBeFilled;
+    private List<Integer> getRandomFrom = new ArrayList<>();
```

Comment:
Not clear what this means

Reply:
is an Array list which contains all the empty grid number available in board, and we are getting a random grid cell number from Among the numbers available in this ArrayList
So, getRandomFrom

Reply:
It should not be named what you've named it. You've named it like a verb. Properties are supposed to be nouns.

What does it contain? That should be it's name.